all-in-one/dockerhub-description.yml

```diff
@@ -0,0 +1,22 @@
+# This uses the peter-evans workflow: https://github.com/peter-evans/dockerhub-description
+name: Update Docker Hub Short-Description & Overview
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - README.md
```

Comment:
Is this the right README to use?

Reply:
This really should be `all-in-one/README.md`. Will make this change.

Reply:
```suggestion
      - all-in-one/BUILDING-TESTING.md
```